thetis/options.py

```diff
@@ -472,6 +472,8 @@ class TracerFieldOptions(FrozenHasTraits):
         None, allow_none=True, help='Source term for the tracer equation')
     diffusivity = FiredrakeScalarExpression(
         None, allow_none=True, help='Diffusion coefficient for the tracer equation')
+    conservative = Bool(
+        False, help='Should the tracer equation be solved in conservative form?')
```

Comment:
maybe use a more descriptive name e.g. `use_conservative_form`, similar to what we use for other boolean options